### DISCUSSION TITLE
The price for each marketing channel changes over time

### DISCUSSION THREAD

**Discussion by openhubber** [google/lightweight_mmm#251] [Q&A]
When the price of each marketing channel changes over time, the clicks (and the later conversions) will also change given the same budget. Right now I pass the price per channel based on model training data, and if the prices for testing time is increased, then the predicted revenue from the MMM model will be higher than the actual revenue. How should we resolve this issue?

**Comment by statm3n** [comment]
Under your described situation, you could choose to use the increased prices for testing time to run the optimization.
